core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -35,25 +36,14 @@ export interface QuestionBackendDict {
 }
 
 export class Question {
-  _id: string;
-  _stateData: State;
-  _languageCode: string;
-  _version: number;
-  _linkedSkillIds: string[];
-  _inapplicableSkillMisconceptionIds: string[];
-
   constructor(
-      id: string, stateData: State, languageCode: string,
-      version: number, linkedSkillIds: string[],
-      inapplicableSkillMisconceptionIds: string[]) {
-    this._id = id;
-    this._stateData = stateData;
-    this._languageCode = languageCode;
-    this._version = version;
-    this._linkedSkillIds = linkedSkillIds;
-    this._inapplicableSkillMisconceptionIds = (
-      inapplicableSkillMisconceptionIds);
-  }
+    private _id: string,
+    private _stateData: State,
+    private _stateDataSchemaVersion: number,
```

Comment:
Why is this added?

Reply:
I added stateDataSchemaVersion because it was present in the QuestionBackendDict but not present in the Question object. I wasn't sure whether it was left out intentionally. Should I remove it?

Reply:
Ah ok. Thanks for explaining. I think it is fine to include it.